### ISSUE TITLE
Long strings are breaking the chat layout

### ISSUE THREAD

**Issue by sintezcs** [carlrobertoh/CodeGPT#75]
`v1.9.0`
Long strings in the output are causing the horizontal scrollbar to appear. It will be better to use soft-wraps.

<img width="574" alt="image" src="https://user-images.githubusercontent.com/2989480/229821564-4f1c9b59-3cdb-4cbe-8753-077ac36a0db2.png">

**Comment by carlrobertoh** [comment]
Hi, thank you for reporting, I'll try to fix this ASAP

**Comment by sintezcs** [comment]
@carlrobertoh Thank you! oh, and also I've just noticed that I do not see the information about my account balance in the top toolbar anymore..

**Comment by carlrobertoh** [comment]
Yeah, OpenAI changed their security policy and we can't request that data anymore.
